modules/openxr/register_types.cpp

```diff
@@ -113,10 +114,19 @@ void initialize_openxr_module(ModuleInitializationLevel p_level) {
 			ERR_FAIL_NULL(openxr_api);
 
 			if (!openxr_api->initialize(Main::get_rendering_driver_name())) {
-				OS::get_singleton()->alert("OpenXR was requested but failed to start.\n"
-										   "Please check if your HMD is connected.\n"
-										   "When using Windows MR please note that WMR only has DirectX support, make sure SteamVR is your default OpenXR runtime.\n"
-										   "Godot will start in normal mode.\n");
+				const char *init_error_message =
+						"OpenXR was requested but failed to start.\n"
+						"Please check if your HMD is connected.\n"
+						"When using Windows MR please note that WMR only has DirectX support, make sure SteamVR is your default OpenXR runtime.\n"
+						"Godot will start in normal mode.\n";
```

Comment:
The message should be wrapped in the tools translate utility (e.g: `TTR("...")`).

Reply:
Note that for `TTR()` to work, it will need to be all on a single line, as the translation strings extraction works line by line and this should all be a single string/paragraph in the translation template.

That being said, I'm not sure we translate this kind of OS alert normally. It's within the realm of error message we want to be easily searchable so it's better if it stays in English, so you can easily find documentation / forum posts / bug reports which mention it.

Reply:
yeah I agree that it doesn't make a whole lot of sense to translate this.